### PR TITLE
Add support for swipe

### PR DESCRIPTION
**Pull request by zenzhu**
Add swipe support according to [XCUIElement swipe method](https://developer.apple.com/reference/xctest/xcuielement#2048660).

### CODE REVIEW

File: WebDriverAgentLib/Commands/FBElementCommands.m [hunk 1]
```diff
@@ -54,6 +54,7 @@ + (NSArray *)routes
     [[FBRoute POST:@"/element/:uuid/click"] respondWithTarget:self action:@selector(handleClick:)],
     [[FBRoute POST:@"/element/:uuid/clear"] respondWithTarget:self action:@selector(handleClear:)],
     [[FBRoute POST:@"/element/:uuid/pinch"] respondWithTarget:self action:@selector(handlePinch:)],
+    [[FBRoute POST:@"/element/:uuid/swipe"] respondWithTarget:self action:@selector(handleSwipe:)],
```
Comment (marekcirkos) [inline WebDriverAgentLib/Commands/FBElementCommands.m]: This custom endpoint please, use `/wda/swipe/` endpoint instead.

Comment (zenzhu) [inline WebDriverAgentLib/Commands/FBElementCommands.m]: I changed it to "uiaElement".

File: WebDriverAgentLib/Commands/FBElementCommands.m [hunk 2]
```diff
@@ -54,6 +54,7 @@ + (NSArray *)routes
     [[FBRoute POST:@"/element/:uuid/click"] respondWithTarget:self action:@selector(handleClick:)],
     [[FBRoute POST:@"/element/:uuid/clear"] respondWithTarget:self action:@selector(handleClear:)],
     [[FBRoute POST:@"/element/:uuid/pinch"] respondWithTarget:self action:@selector(handlePinch:)],
+    [[FBRoute POST:@"/uiaElement/:uuid/swipe"] respondWithTarget:self action:@selector(handleSwipe:)],
```
Comment (marekcirkos) [inline WebDriverAgentLib/Commands/FBElementCommands.m]: No no, this is legacy stuff, that I want to kill :). Please use `/wda/..`

Comment (zenzhu) [inline WebDriverAgentLib/Commands/FBElementCommands.m]: Fixed :).

**Pull request closed without merging**